zsh-syntax-highlighting.zsh

```diff
@@ -28,13 +29,17 @@
 # -------------------------------------------------------------------------------------------------
 
 # First of all, ensure predictable parsing.
-zsh_highlight__aliases=`builtin alias -Lm '[^+]*'`
+local zsh_highlight__aliases="`builtin alias -Lm '[^+]*'`"
```

Comment:
Since you're touching this line anyway?, can you convert backticks to `$( )`?

Reply:
Touché.

Reply:
Err, UI bug. I meant to say: Sure can; what's the rationale?

Reply:
Consistency with the rest of the code and `$( )` nests better. I generally replace any backticks I come across.

Reply:
I'm not as zealous, but sure, I'll have it use the house style.

Reply:
Done.